tendrl/monitoring_integration/upgrades/delete_dashboards.py

```diff
@@ -99,15 +99,19 @@ def main():
         username = config.get('security', 'admin_user')
         password = config.get('security', 'admin_password')
         default_ip = "127.0.0.1"
-
+        default_port = 3000
```

Comment:
Are we always using default port only?

Reply:
For now, we are using 3000 port as default, We don't have proper documentation to change the port number. Because it will affect a lot of places. including reverse proxy file.

Reply:
Ack